sdks/python/apache_beam/examples/wordcount_xlang_sql.py

```diff
@@ -101,12 +89,33 @@ def main():
   # workflow rely on global context (e.g., a module imported at module level).
   pipeline_options.view_as(SetupOptions).save_main_session = True
 
-  p = beam.Pipeline(options=pipeline_options)
-  # Preemptively start due to BEAM-6666.
-  p.runner.create_job_service(pipeline_options)
+  with beam.Pipeline(options=pipeline_options) as p:
+    if isinstance(p.runner, portable_runner.PortableRunner):
```

Comment:
Can we generalize this so that users can easily try this out with Dataflow as well ?

Reply:
This will just work for Dataflow. That's why I added the guard.